CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# How to contribute
+
+## Voter dApp
+
+This is a browser client that UMA token holders can use to vote on price requests. The dApp interacts with the deployed [DVM](https://docs.umaproject.org/uma/oracle/technical_architecture.html) code using MetaMask's browser extension. The source code was bootstrapped from [Create React App](https://github.com/facebook/create-react-app).
+
+Features currently include:
+
+- Deploy a [designated voting proxy contract](https://docs.umaproject.org/uma/oracle/voting_with_UMA_2-key_contract.html)
+- Commit an encrypted vote
+- Backup salts associated with latest encrypted votes
+- Reveal an encrypted vote
+- Retrieve rewards for resolved votes
+- View results of resolved votes
+
+### Making changes to the Voter dApp
+
+The source code for this React app can be found in `./voter-dapp/src/`. At a minimum, you must ensure that a user can go through a simple commit and reveal flow, both using a voting proxy and not using one.
+
+#### Steps to generate price requests
+
+0. Change your directory to `./protocol/`
+1. Start a local blockchain on port `9545` with enough starting ETH balances and a high enough gas limit:
+   ```sh
+   ganache-cli -p 9545 -e 10000000000 -l 9000000
+   ```
+   This seeds every account with `10000000000` ETH and sets the block gas limit to 9 million wei.
+1. Open another window and deploy all contracts:
+   ```sh
```

Comment:
small nit: we normally have represented stuff that we run in the command line in the docs with the `bash` flag, like this:
```suggestion
   ```bash
```

Reply:
cool will do